tests/pricing_models/yieldspace_test_calc_out_given_in.py

```diff
@@ -206,12 +206,12 @@ def test_calc_out_given_in_success(self):
                     # We want to solve for the amount of bonds out given the
                     # amount of shares coming in, so we set up the problem as:
                     #
-                    #   k = (c / μ) * (μ * (z + d_z)) ** (1 - t) + (2 * y + c * z - d_y') ** (1 - t)
-                    #     = 110_000 ** (1 - T) + (300_000 - d_y') ** (1 - T)
+                    #   k = (c / mu) * (mu * (z + d_z)) ** (1 - tau) + (2 * y + c * z - d_y') ** (1 - tau)
+                    #     = 110_000 ** () + (300_000 - d_y') ** (1 - tau)
```

Comment:
I'd recommend that you look through the diffs and find all cases in which `(1 - T)` => `(1 - tau)` and all other changes like this.